src/gui/configgui.h.in

```diff
@@ -0,0 +1,4 @@
+#ifndef CONFIG_GUI_H
```

Comment:
@mgallien I had to introduce it because config.h.in is generated by a different CMake file and it has no access to variables inside the CMake file inside the GUI folder.

Reply:
@mgallien I tried your suggestion. It doesn't work.